transform/variant/requirements.txt

```diff
@@ -0,0 +1,9 @@
+pytest==3.6.3
+pandas==0.23.3
+mygene==3.0.0
+myvariant==0.3.1
+requests==2.19.1
+requests-cache==0.4.13
+# -e git+git@github.com:bmeg/bmeg-etl.git@33e27b25f603ef8847b089415c949ada1332a648#egg=bmeg_etl
```

Comment:
This is too specific.

Reply:
The version numbers? or, the git reference?

Reply:
The git reference is going to be out of date almost immediately. Maybe just reference bmeg-etl without the commit id?